examples/seismic/tti/wavesolver.py

```diff
@@ -34,6 +34,12 @@ def __init__(self, model, geometry, space_order=4, kernel='centered',
         self.geometry = geometry
         self.kernel = kernel
 
+        if model.fs and kernel == 'staggered':
+            warning("Bad arguments in AnisotropicWaveSolver() - " +
+                    "if model.fs is True, kernel should be centered. " +
+                    "Switching to 'centered'")
+            self.kernel = 'centered'
```

Comment:
It should raise an error not change user set parameters

Reply:
Ok, I will use `raise ValueError` then.